code/cfile/cfile.cpp

```diff
@@ -863,7 +876,12 @@ int cfget_cfile_block()
 
 	// If we've reached this point, a free Cfile_block could not be found
 	nprintf(("Warning","A free Cfile_block could not be found.\n"));
-	Assert(0);	// out of free cfile blocks
+
+	// Dump a list of all opened files
+	mprintf(("Out of cfile blocks! Currently opened files:\n"));
+
```

Comment:
:facepalm: I added the dump function and forgot to readd it here. I'll fix it in along with the other issue you found.